__tests__/index.test.js

```diff
@@ -231,6 +231,31 @@ describe("/api/articles/:article_id/comments", () => {
   });
 });
 
+
+describe("/api/users", () => {
+  test("GET:200 sends an array of users objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const arr = body.users;
+        expect(arr.length).toBe(4);
+        arr.forEach((user) => {
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
+      });
+  });
+  test("GET:404 responds with an error message saying path is not found", () => {
```

Comment:
This test isn't specific to this endpoint. It's just generic to any incorrect endpoint after api/. Please either remove, or pop it into a different describe block 👍